components/demo/demo-snippet.js

```diff
@@ -24,8 +24,9 @@ class DemoSnippet extends LitElement {
 	}
 
 	render() {
+		const dirAttr = this._dir === 'rtl' ? 'rtl' : 'ltr';
 		return html`
-			<div class="d2l-demo-snippet-demo" dir="${this._dir}">
+			<div class="d2l-demo-snippet-demo" dir="${dirAttr}">
```

Comment:
It was complaining that `dir` can't be any arbitrary string -- so forcing it to be one of these values fixes things.